packages/vhd-lib/src/createReadableSparseStream.js

```diff
@@ -25,19 +25,18 @@ const VHD_BLOCK_SIZE_SECTORS = VHD_BLOCK_SIZE_BYTES / SECTOR_SIZE
 function createBAT(
   firstBlockPosition,
   fragmentLogicAddressList,
+  fragmentSize,
   ratio,
   bat,
   bitmapSize
 ) {
```

Comment:
Too many positional params, used named params instead:

```suggestion
function createBAT({
  firstBlockPosition,
  fragmentLogicAddressList,
  fragmentSize,
  ratio,
  bat,
  bitmapSize,
}) {
```